Islandora-Devops/isle-dc
# Tls

Right now I'm only exposing Drupal, Cantaloupe, and Matomo over https as those are the what would be exposed in a minimal production environment.  Other services are available over regular http, but to change that, you just need to copy/paste/adapt some traefik labels.

To test:
- Pull down https://github.com/Islandora-Devops/isle-buildkit/pull/66 and build the images
- Pull down this PR into your isle-dc repo
- rm docker-compose.yml docker-compose.env.yml .env` to make sure everything is clean.
- `cp sample.env .env`
- Edit `.env` so that `REPOSITORY` is set to `local` so you get the images you just built
- `make`, which will download the default certs and regenerate your files.
- `docker-compose up -d`

Your site should be available at `https://islandora-isle-dc.traefik.me`.

If you are using a remote instance with a domain you own, or an instance with an IP but no domain, follow the instructions in the README to set the proper configuration in your `.env` and `docker-compose.env.yml` files
